### PR TITLE
fix biocore/qiita#1681

### PR DESCRIPTION
**Pull request by antgonza**
fix biocore/qiita#1681. 

@ElDeveloper or @josenavas could you review? Thanks!

### CODE REVIEW

**Comment by josenavas** [comment]
👍

File: qp_target_gene/split_libraries/split_libraries_fastq.py [hunk 1]
```diff
@@ -131,7 +131,17 @@ def generate_per_sample_fastq_command(forward_seqs, reverse_seqs, barcode_fps,
             samples.append(sn_by_rp[f])
             del sn_by_rp[f]
         except KeyError:
-            pass
+            # if we get to this point it's possible that we removed
+            # unnecessarily the prefix number of the sample names so let's
+            # try again without that step
+            f = basename(fname)
+            if 'fastq' in f.lower().rsplit('.', 2):
```
Comment (ElDeveloper) [inline qp_target_gene/split_libraries/split_libraries_fastq.py]: Shouldn't you use splitext?

Comment (ElDeveloper) [inline qp_target_gene/split_libraries/split_libraries_fastq.py]: Disregard this comment. ⌛

**Comment by ElDeveloper** [comment]
👍

**Pull request merged**